File: problemas/1000/1018.cs
using System;

namespace uri1018
{
    internal static class Program
    {
        private static void Main()
        {
            var cedulas = new int[7] { 100, 50, 20, 10, 5, 2, 1 };
            int valor = int.Parse(Console.ReadLine());
            Console.WriteLine(valor);

            foreach(int cedula in cedulas)
            {
                Console.WriteLine($"{valor / cedula} nota(s) de R$ {cedula},00");
                valor %= cedula;
            }
        }
    }
}

File: problemas/1000/1018.php
<?php

$cedulas = [100, 50, 20, 10, 5, 2, 1];

$valor = readline();
echo($valor . "\n");

foreach($cedulas as $cedula) {
    $quantidade = intdiv($valor, $cedula);
    echo("{$quantidade} nota(s) de R$ {$cedula},00\n");
    $valor %= $cedula;
}